docs/ql-style-guide.md

```diff
@@ -209,6 +211,9 @@ class Type extends ... {
 
   /** ... */
   Type getATypeParameter() { ... }
+  
+  /** ... */
+  predicate getHttpConnection() { ... }
```

Comment:
Possibly add `getSsaVariable`? For me, SSA somehow seems like the canonical example of this.

Reply:
Done :+1: